md5.js

```diff
@@ -149,35 +148,34 @@ return state;
  * 8-bit unsigned value arrays.
  */
 function md5blk(s) { /* I figured global was faster.   */
-var md5blks = [], i; /* Andy King said do it this way. */
-for (i=0; i<64; i+=4) {
-md5blks[i>>2] = s.charCodeAt(i)
-+ (s.charCodeAt(i+1) << 8)
-+ (s.charCodeAt(i+2) << 16)
-+ (s.charCodeAt(i+3) << 24);
-}
-return md5blks;
+  var md5blks = [], i; /* Andy King said do it this way. */
+  for (i=0; i<64; i+=4) {
+    md5blks[i>>2] = s.charCodeAt(i)
+    + (s.charCodeAt(i+1) << 8)
+    + (s.charCodeAt(i+2) << 16)
+    + (s.charCodeAt(i+3) << 24);
+  }
+  return md5blks;
 }
 
 var hex_chr = '0123456789abcdef'.split('');
 
-function rhex(n)
-{
-var s='', j=0;
-for(; j<4; j++)
-s += hex_chr[(n >> (j * 8 + 4)) & 0x0F]
-+ hex_chr[(n >> (j * 8)) & 0x0F];
-return s;
+function rhex(n) {
+  var s='', j=0;
+  for(; j<4; j++)
+  s += hex_chr[(n >> (j * 8 + 4)) & 0x0F]
+  + hex_chr[(n >> (j * 8)) & 0x0F];
```

Comment:
Identifier 'hex_chr' is not in camel case.<br>Bad line breaking before '+'.<br>Unexpected use of '>>'.<br>Unexpected use of '&'.